.github/workflows/codeql.yml

```diff
@@ -39,13 +39,24 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
 
+    # Check if Python files are changed
+    - name: Check for Python file changes
+      id: python_check
+      run: |
+        git fetch origin ${{ github.base_ref }}
+        git diff --name-only origin/${{ github.base_ref }}...${{ github.head_ref }} | grep '\.py$' || echo "no-python-changes"
```

Comment:
this would exclude changes to `python/bin/suricatasc` and `python/bin/suricatactl`, right? For me it would make sense to simply check if there was any diff in the whole `python/` dir.